q2_feature_table/plugin_setup.py

```diff
@@ -85,6 +107,14 @@
     parameters={},
     outputs=[
         ('merged_data', FeatureData[Sequence])],
+    input_descriptions={
+        'data1': 'The first collection of feature sequences to be merged.',
+        'data2': 'The second collection of feature sequences to be merged.',
+    },
+    parameter_descriptions={},
+    output_descriptions={
+        'merged_data': 'The resulting collection of feature sequences.'
```

Comment:
*The resulting collection of feature sequences containing all feature sequences from `data1` and `data2`.*